.github/workflows/release_charts.yml

```diff
@@ -38,6 +38,7 @@ jobs:
       - name: Install additional Helm repos
         run: |
           helm repo add bitnami https://charts.bitnami.com/bitnami
+          helm repo add bevel https://sownak.github.io/bevel
```

Comment:
Do we require this helm repo ?

Reply:
Yes for now, will have to be updated to Hyperledger once all charts are done